app/models/follow.rb

```diff
@@ -0,0 +1,31 @@
+class Follow < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Because of that we do not have implemented it. Yes, maybe is a YAGNI thing for now.